### ISSUE TITLE
Docker entrypoint errors when calling commands outside init directory

### ISSUE THREAD

**Issue by mcapuccini** [kubenow/KubeNow#305]
When calling commands outside the init directory the Docker entrypoint errors:

```
$ kn openstack --version
Using KubeNow version: 0.3.0rc3
docker: Error response from daemon: oci runtime error: container_linux.go:265: starting container process caused "exec: \"/KubeNow_root/bin/docker-entrypoint\": stat /KubeNow_root/bin/docker-entrypoint: no such file or directory".
``` 

```
$ kn gcloud --version
Using KubeNow version: 0.3.0rc3
docker: Error response from daemon: oci runtime error: container_linux.go:265: starting container process caused "exec: \"/KubeNow_root/bin/docker-entrypoint\": stat /KubeNow_root/bin/docker-entrypoint: no such file or directory".
```

We should move the test for these commands before initing the deployment in Travis, to avoid errors in the future. @andersla could you give it a look?

**Comment by mcapuccini** [comment]
No problem from inside the directory:

```
vpnpool188-189:test marcocapuccini$ kn gcloud --version
Using KubeNow version: 0.3.0rc3
Google Cloud SDK 179.0.0
alpha 2017.11.06
beta 2017.11.06
bq 2.0.27
core 2017.11.06
gsutil 4.28
vpnpool188-189:test marcocapuccini$ kn openstack --version
Using KubeNow version: 0.3.0rc3
openstack 3.11.0
```

**Comment by mcapuccini** [comment]
Fixed #306
